.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
+
+**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**
+
+## Documentation
+
+- [ ] `docs/` have been added/updated **OR**
+- [ ] No updates to the docs is considered unnecessary after careful consideration.
+
+## QC
+
+- [ ] `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
```

Comment:
## Fixes #361?
In the OP it says it fixes this issue. The issue is related. But it does not seem to address the core concern of the issue, which involves running a `.PHONY` goal to represent the changes.

_361 says:_
> Requirement in PR .github template. Run the .PHONY goal associated w/ your change in fast mode.
> 
> For example:
> Identify the .PHONY goal associated with an issue. Run that goal with IMP=false MIR=false. Should not take 18 hours to run. Maybe like up to 2 hours max.

I don't think we can have a phony goal for every change. And even when we do have a phony goal, I think we've already overridden this requirement by requiring that `build-mondo-ingest` be ran in exception instead.

Although we could also add a 3rd option 'c' to the "QC" step:
a. `build-mondo-ingest`
b. _No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary._
c. Run just a single goal that represents the changes

Reply:
Good point. Do you think the changes to the pipeline and the running of the goal that actually enacts the fix should be done on the same PR? Maybe discuss with @twhetzel and see what she thinks! I could go multiple ways here.

Reply:
The choice here seems fraught with trepidation and there are several options. Whatever we decide, I think, should be codified in:
- #405 

This is what I personally think is the best option. I think this is what Trish and I came up with:
1. Feature -> Develop
  - Run option 'c', _ a single goal that represents the changes_
2. Develop -> Master
  - `build-mondo-ingest`

Reply:
This sounds very reasonable!

Reply:
I agree that this does not seem to fix #361 related to running a .PHONY goal. 

The plan Joe outlined above is what we discussed before, but in some other PR conversation or Slack Nico was hesitant about this plan so I'm not sure what has changed now.

Reply:
I personally prefer to always run the whole pipeline, but I am also fine with @joeflack4's reduced solution as long as there _is_ a development branch that is never merged into `main` before the the whole pipeline is run. But @twhetzel feel free to make a call, I dont mind other way. I always run the whole pipeline when I make changes, but I can see that our main developers may not be too happy :P

Reply:
Ah, your comment is referring to "run the whole pipeline" - yes, given the shorter time now, yes let's just make sure the entire pipeline is run regardless of any future branching strategy.